src/main/scala/io/pubcrawler/stout/db/Schemas.scala

```diff
@@ -0,0 +1,82 @@
+package io.pubcrawler.stout.db
+
+import slick.jdbc.PostgresProfile.api._
+import java.sql.Date
+import java.sql.Time
+
+
+class Stop(tag: Tag) extends Table[(Int, String, String, Float, Float)](tag, "STOP") {
+  def id = column[Int]("STOP_ID", O.PrimaryKey)
+  def title = column[String]("TITLE")
+  def address = column[String]("ADDRESS")
+  def lat = column[Float]("LAT")
+  def lng = column[Float]("LNG")
+  def * = (id, title, address, lat, lng)
+}
+
+class Route(tag: Tag) extends Table[(Int, Int)](tag, "ROUTE") {
+  def id = column[Int]("ROUTE_ID", O.PrimaryKey)
+  def ownerId = column[Int]("OWNER_ID")
+  def * = (id, ownerId)
+}
+
+class RouteStop(tag: Tag, stops: TableQuery[Stop], routes: TableQuery[Route]) extends Table[(Int, Int, Int)](tag, "ROUTE_STOP") {
+  def routeId = column[Int]("ROUTE_STOP_ID")
+  def stopId = column[Int]("STOP_ID")
+  def order = column[Int]("ORDER")
+  def * = (routeId, stopId, order)
+
+  def stop = foreignKey("STOP_FK", stopId, stops)(_.id)
+  def route = foreignKey("ROUTE_FK", routeId, routes)(_.id)
+}
+
+class Crawl(tag: Tag, stops: TableQuery[Stop]) extends Table[(Int, String, Int, Time, Float, Float, Int, String, String)](tag, "CRAWL") {
+  def id = column[Int]("CRAWL_ID", O.PrimaryKey)
+  def title = column[String]("TITLE")
+  def ownerId = column[Int]("OWNER_ID")
+  def dateTime = column[Time]("DATE_TIME")
+  def lat = column[Float]("LAT")
+  def lng = column[Float]("LNG")
+  def radius = column[Int]("RADIUS")
+  def city = column[String]("CITY")
+  def description = column[String]("DESCRIPTION")
+  def * = (id, title, ownerId, dateTime, lat, lng, radius, city, description)
+}
+
+class CrawlParticipants(tag: Tag) extends Table[(Int, Int, Boolean)](tag, "CRAWL_PARTICIPANTS") {
+  def crawlId = column[Int]("CRAWL_ID")
+  def userId = column[Int]("USER_ID")
+  def status = column[Boolean]("STATUS")
```

Comment:
I was taking notes from the docs, but I'll look into enums, and I wholeheartedly it reads better with enums.

http://slick.lightbend.com/doc/3.1.1/schemas.html#table-rows